### ISSUE TITLE
Only appears after page refresh when using Turbo

### ISSUE THREAD

**Issue by taylorthurlow** [julienbourdeau/debugbar#9]
When using [turbo](https://github.com/hotwired/turbo-rails), the "drive" component of turbo is preventing the Javascript from loading properly and therefore debugbar only loads if you refresh a page. Regular navigation does not cause it to pop up. It works fine when I disable Turbo with:

```javascript
// app/javascript/application.js

import { Turbo } from "@hotwired/turbo-rails"
Turbo.session.drive = false
```

I think, in theory you could duplicate the whichever event listener normally triggers the rendering of the bar and also trigger it on the `turbo:load` event. I tried playing with the JS but it's tough with the obfuscated JS.

**Comment by julienbourdeau** [comment]
I haven't got into turbo and hotwire yet. Thank you for the report and the details about `Turbo.session.drive`. I'll have a look quickly.

**Comment by jbrodie** [comment]
Can confirm this.
Wasn't sure how it would work with Turbo, so it is good to know that this will be addressed.

**Comment by Riddlerrr** [comment]
I can confirm it as well.
Let me know if I can help to debug or fix it.

**Comment by hss-mateus** [comment]
Fixed it by wrapping it in a div with `data-turbo-permanent="true"`

```erb
<div data-turbo-permanent="true" id="debugbar">
  <%= debugbar_javascript if defined?(Debugbar) %>
</div>
```

**Comment by nithinbekal** [comment]
👋 I've proposed a fix for this in https://github.com/julienbourdeau/debugbar/pull/25.

**Comment by julienbourdeau** [comment]
I believe this was fixed in v0.3.0. There is a breaking change to load the js file directly in the `head`. Please let me know how it works for you 🙏

```diff
<!DOCTYPE html>
<html>
  <head>
    ... 
+    <%= debugbar_head %>
  </head>
    
  <body>
    ...
-    <%= debugbar_javascript %>
+    <%= debugbar_body %>
  </body>
</html>
```
